components/ledger/.gitignore

```diff
@@ -15,5 +15,4 @@ ledger.test
 antlr-*-complete.jar
 go.work
 go.work.sum
-ledger
```

Comment:
Why did you remove that ? it should prevent us to push the binary, am I right ?

Reply:
yes, but it was hide the "ledger" package ^^
This has probably been added by error.